synapse/lib/stormtypes.py

```diff
@@ -2128,6 +2127,9 @@ async def wput(self, sha256, url, headers=None, params=None, method='PUT', ssl=T
         params = self.strify(params)
         headers = self.strify(headers)
 
+        if proxy is not None:
+            self.runt.confirm(('axon', 'proxy'))
```

Comment:
Is this a permission we need to be checking on the AxonApi too?

Reply:
Yes, good catch, thanks! :)